src/pylogctx/django.py

```diff
@@ -1,16 +1,16 @@
 from __future__ import absolute_import
 
 import logging
-
 from django.conf import settings
-
 from pylogctx import context, AdapterNotFound
-
+try:
+    from django.utils.deprecation import MiddlewareMixin
+except ImportError:
+    MiddlewareMixin = object
 
 logger = logging.getLogger(__name__)
 
-
-class OuterMiddleware(object):
+class OuterMiddleware(MiddlewareMixin):
```

Comment:
Can you add a blank line just before the class definition so that the flake8 tests succeed? (output: https://travis-ci.org/peopledoc/pylogctx/builds/370252289#L509)

Reply:
Of course, not sure if I did it correctly though, but it seems the commit ended up in the right spot, and that it triggered a new build.

Reply:
Thanks!